var/spack/repos/builtin/packages/libsmeagol/package.py

```diff
@@ -0,0 +1,66 @@
+# Copyright Spack Project Developers. See COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class Libsmeagol(MakefilePackage):
+    """Non-equilibrium Green's function (NEGS) code."""
+
+    homepage = "https://github.com/StefanoSanvitoGroup/libsmeagol"
+    git = "https://github.com/StefanoSanvitoGroup/libsmeagol.git"
+
+    maintainers("RMeli")
+
+    license("GPL-2.0-or-later", checked_by="RMeli")
+
+    version("main", branch="main")
+    version("1.2", commit="fefed1bb4fceca584c3014debb169e8ed4ce1289")
```

Comment:
Do you think it'd be worth using the `libsmeagol-1.2` branch? I don't know if it's intended to be updated as a release branch, but perhaps a commit from there would be semantically more correct? In practice it doesn't seem to matter too much, the diff between the `fefed1...` that you have here and the current `libsmeagol-1.2` branch is only cosmetic: https://github.com/StefanoSanvitoGroup/libsmeagol/compare/libsmeagol-1.2...fefed1bb4fceca584c3014debb169e8ed4ce1289.

Reply:
I think it's fine with the commit that you have, just wanted to check. The commit here can after all still be changed in the future if there's a more established release process.